all.json

```diff
@@ -6619,25 +6619,16 @@
 		"pancakeswap.finance.alifdashproject.com",
 		"pancakeswap.finance.biswap2.com",
 		"pancakeswap.finance.bitminer-cloud.com",
-		"pancakeswap.finance.cake-token.in",
-		"pancakeswap.finance.fastservicetrack.com",
-		"pancakeswap.finance.goodsservicedogs.com",
-		"pancakeswap.finance.goswap.fastservicetrack.com",
+		"fastservicetrack.com",
+		"goodsservicedogs.com",
```

Comment:
Multiple hist - site now defunct. Only mention on google does point back to this list, nowhere else.